howtos/manage_shared_libraries/rpaths.rst

```diff
@@ -105,8 +105,9 @@ Remember to pass the ``KEEP_RPATHS`` variable to the ``conan_basic_setup``:
     include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
     conan_basic_setup(KEEP_RPATHS)
 
-Then, you could, for example, use the ``@executable_path`` in OSX and ``$ORIGIN`` in Linux  to adjust
-a relative path from the executable:
+Then, you could, for example, use the ``@executable_path`` in OSX and ``$ORIGIN`` in Linux  to adjust a relative path from the executable.
+Also, enabling ``CMAKE_BUILD_WITH_INSTALL_RPATH`` will make the paths **always relative** to the application being built and the executable
```

Comment:
`CMAKE_BUILD_WITH_INSTALL_RPATH` doesn't make the paths "always relative". It uses CMAKE_INSTALL_RPATH` declared paths also for building (without need of doing cmake install later).
I would move that sentence as a clarification after the code, something like:

```
Remember to activate `CMAKE_BUILD_WITH_INSTALL_RPATH` if you want to use the `CMAKE_INSTALL_RPATH` and you are not using cmake install process.
```